include/evmc/evmc.h

```diff
@@ -70,7 +70,8 @@ enum evmc_call_kind
     EVMC_CALL = 0,         /**< Request CALL. */
     EVMC_DELEGATECALL = 1, /**< Request DELEGATECALL. The value param ignored. */
     EVMC_CALLCODE = 2,     /**< Request CALLCODE. */
-    EVMC_CREATE = 3        /**< Request CREATE. Semantic of some params changes. */
+    EVMC_CREATE = 3,       /**< Request CREATE. Semantic of some params changes. */
```

Comment:
You can remove the "Semantic of some params changes" part in both cases.

Reply:
I think these should state which `evmc_revision` they are valid in (frontier, homestead, byzantium, constantinople).